From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Tue, 22 Dec 2020 04:05:53 +0000
Subject: ANDROID: db845c_gki.fragment make sure MSM_PINCTRL=m

The MSM_PINCTRL is no longer a selected option but a dependency,
so we need to add the config explictly in the fragment.

Fixes: be117ca32261 ("pinctrl: qcom: Kconfig: Rework PINCTRL_MSM to be a depenency rather then a selected config")
Signed-off-by: John Stultz <john.stultz@linaro.org>
Change-Id: Ib77f8d645b33c3879afe9e9e4cdf177eecf04b31
---
 arch/arm64/configs/db845c_gki.fragment | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -9,6 +9,7 @@ CONFIG_SERIAL_QCOM_GENI=m
 CONFIG_SERIAL_QCOM_GENI_CONSOLE=y
 CONFIG_I2C_QCOM_GENI=m
 CONFIG_I2C_QUP=m
+CONFIG_PINCTRL_MSM=m
 CONFIG_PINCTRL_QCOM_SPMI_PMIC=m
 CONFIG_PINCTRL_SDM845=m
 CONFIG_POWER_RESET_QCOM_PON=m
